### ISSUE TITLE
CRITICAL [UX Reliability] Tracking issue for random unreads in LHN

### ISSUE THREAD

**Issue by muttmuure** [Expensify/App#44683]
## Problem

We're seeing a lot of transaction or report threads being made unread in the LHN  - we need to deep dive into why that is and fix it. 

# Solution 

Let's track all of the reports of unexpected unreads in the LHN in this issue and keep this issue open until we get to the bottom of it and fix all instances of this bug.

**Comment by mallenexpensify** [comment]
@muttmuure , do you want to link them in the OP or are the mentions above _enough_ for tracking purposes?

**Comment by muttmuure** [comment]
I think the mentions are fine

**Comment by muttmuure** [comment]
No update yet - we're working through them

**Comment by mallenexpensify** [comment]
[New chats not loading or showing in LHN](https://github.com/Expensify/App/issues/45075#top) 
@muttmuure do you _want_ the above added here?  It might be the opposite, where unread chats **aren't** showing.

**Comment by puneetlath** [comment]
So we've got a few known issues that we're tackling: 
- Better handling for the "pre-existing report" case: https://github.com/Expensify/App/issues/45035
- Issue where exporting a report as a copilot marks it as unread when it shouldn't: https://github.com/Expensify/App/issues/44693
- Filtering payroll and bill reports from the LHN: https://github.com/Expensify/App/issues/45104
- Archived chats showing up as unread: https://expensify.slack.com/archives/C05LX9D6E07/p1720624906101939?thread_ts=1720290639.217409&cid=C05LX9D6E07

**Comment by deetergp** [comment]
Investigations are ongoing.

**Comment by muttmuure** [comment]
Latest update was here: https://expensify.slack.com/archives/C05LX9D6E07/p1720799600873919

**Comment by deetergp** [comment]
The PR to prevent sending updates to `paycheck` and `bill` reports was approved and merged on Tuesday afternoon. @puneetlath and I identified [another situation](https://expensify.slack.com/archives/C05LX9D6E07/p1721161095860129?thread_ts=1721157878.383419&cid=C05LX9D6E07) where we might be incorrectly marking reports as unread. I'm hoping to have a PR to address that raised later in the day today.

**Comment by deetergp** [comment]
[This PR](https://github.com/Expensify/Auth/pull/11700) just went out to Auth which catches a number of edge cases around reports becoming unexpectedly unread when they receive ESM supported system messages.

**Comment by muttmuure** [comment]
@deetergp it seems like we've made some progress on this. 

It looks like this is the only issue left with random unreads? https://github.com/Expensify/App/issues/44693

**Comment by muttmuure** [comment]
Can you give an update on the status of the clean up? Thank you!

**Comment by deetergp** [comment]
@muttmuure I spent some time on it on Thursday and was able to reproduce this locally with bulk exporting. I was OOO on Friday and am back and going to dig into it further today to see if I can prove or disprove my commit conflict theory.

**Comment by deetergp** [comment]
I found a pretty big bug where, on reports where multiple users should be receiving updated last read times, we were only ever sending one for the first accountID returned by [the query](https://github.com/Expensify/Auth/blob/4f77faff0d90b0d90cc9f1968432cf94c9416c03/auth/lib/Report.cpp#L11256-L11281). That went to production around 6:30 AM PDT on the 30th. @puneetlath and I did some testing yesterday and found that we should be including `REIMBURSED` in the list of ESM supported actions. I made a PR for it that was merged earlier today, but has not yet been deployed to production. @twisterdotcom opened [a GH](https://github.com/Expensify/Expensify/issues/416602) earlier today about reports going unread because of  POLICYCHANGELOG actions being added, but each one of them happened before the big bugfix went out. I've left the GH open and made it weekly since we are in a wait-and-see pattern at this point.
